client/src/components/DataFiles/DataFilesModals/DataFilesAddProjectModal.js

```diff
@@ -89,6 +90,7 @@ const DataFilesAddProjectModal = () => {
         isOpen={isOpen}
         toggle={toggle}
         className="dataFilesModal"
+        styleName="root"
```

Comment:
__Revisit After Change__

[These styles might be removed from `DataFilesAddProjectModal.module.scss`.](https://github.com/TACC/Frontera-Portal/pull/311/files#r562196906)